### PR TITLE
Support {find, update, delete} graph annotation(s)

### PR DESCRIPTION
**Pull request by syou6162**
ref: https://github.com/mackerelio/mackerel-client-go/pull/39

Implemented these methods for graph annotations.

- `GET /api/v0/graph-annotations`
- `PUT /api/v0/graph-annotations/<annotationId>`
- `DELETE /api/v0/graph-annotations/<annotationId>`

### CODE REVIEW

File: graph_annotation.go [hunk 1]
```diff
@@ -16,3 +25,74 @@ func (c *Client) CreateGraphAnnotation(annotation *GraphAnnotation) error {
...
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	resp, err := c.Request(req)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	var annotation = GraphAnnotation{}
```
Comment (itchyny) [inline graph_annotation.go]: [nit] `var annotation GraphAnnotation` is preferred when this line indicates the allocation (not initialization of empty annotation) IMO.

File: graph_annotation_test.go [hunk 2]
```diff
@@ -74,3 +74,141 @@ func TestCreateGraphAnnotation(t *testing.T) {
...
+	client, _ := NewClientWithOptions("dummy-key", ts.URL, false)
+	annotations, err := client.FindGraphAnnotations("My-Blog", 1485675275, 1485675299)
+
+	if err != nil {
+		t.Error("err shoud be nil but: ", err)
+	}
+
+	if annotations[0].Service != "My-Blog" {
+		t.Error("request sends json including Service but: ", annotations[0].Service)
+	}
```
Comment (itchyny) [inline graph_annotation_test.go]: Test `roles`, `from` and `to` if possible

File: graph_annotation_test.go [hunk 3]
```diff
@@ -74,3 +74,141 @@ func TestCreateGraphAnnotation(t *testing.T) {
...
+
+	client, _ := NewClientWithOptions("dummy-key", ts.URL, false)
+	annotation, err := client.DeleteGraphAnnotation("123456789")
+
+	if err != nil {
+		t.Error("err shoud be nil but: ", err)
+	}
+	if annotation.Service != "My-Blog" {
+		t.Error("request sends json including Service but: ", annotation.Service)
+	}
```
Comment (itchyny) [inline graph_annotation_test.go]: Test roles, description if possible.

**Comment by itchyny** [comment]
LGTM

**Comment by syou6162** [comment]
Thanks!

**Pull request merged**